site/content/en/docs/Advanced/allocator-service.md

```diff
@@ -75,44 +82,40 @@ spec:
     kind: ClusterIssuer
 EOF
 
-# Optional: Store the secret ca.crt in a file to be used by the client for the server authentication
-TLS_CA_FILE=ca.crt
-TLS_CA_VALUE=`kubectl get secret allocator-tls -n agones-system -ojsonpath='{.data.ca\.crt}'`
-echo ${TLS_CA_VALUE} | base64 -d > ${TLS_CA_FILE}
-
-# In case of MacOS
-# echo ${TLS_CA_VALUE} | base64 -D > ${TLS_CA_FILE}
-
 # Add ca.crt to the allocator-tls-ca Secret
+TLS_CA_VALUE=`kubectl get secret allocator-tls -n agones-system -ojsonpath='{.data.ca\.crt}'`
```

Comment:
```suggestion
TLS_CA_VALUE=$(kubectl get secret allocator-tls -n agones-system -ojsonpath='{.data.ca\.crt}')
```

Since we're in here making changes.